corehq/toggles/__init__.py

```diff
@@ -2631,6 +2631,14 @@ def _handle_geospatial_es_index(domain, is_enabled):
 
 )
 
+CAMPAIGN_DASHBOARD = StaticToggle(
+    'campaign_dashboard',
+    'Allows access to Campaign Dashboard',
+    TAG_SOLUTIONS_LIMITED,
+    namespaces=[NAMESPACE_DOMAIN],
+    description='Access to new dashboard being developed to show various metrics for campaigns',
```

Comment:
I'd tweak the wording slightly so that it's still true after April 3.
```suggestion
    description='Access to a dashboard to show various metrics for campaigns',
```

Reply:
Sounds good @kaapstorm , updated.